### ISSUE TITLE
Not detecting any outliers on purely categorical datasets

### ISSUE THREAD

**Issue by ToxicFyre** [david-cortes/outliertree#4]
[outliertree_test.zip](https://github.com/david-cortes/outliertree/files/6251302/outliertree_test.zip)

Hello, I am comparing some outlier detectors on purely categorical datasets, but whenever I run OutlierTree on purely categorical datasets it doesn't return any outliers (With some exceptions). Is there any different parameterization that you recommend? 

I attached a sample of what I am using to test it, so you can see what I mean. 

Cheers and thanks for the help,
ToxicFyre

**Comment by david-cortes** [comment]
Indeed, if you look at the parameters, it has something about minimum sizes of branches to split. Apart from that, categorical outliers are just harder to identify. Ideas for criteria are welcome.

**Comment by ToxicFyre** [comment]
Thank you for your reply. Yes, I noticed that having a few ordinal and numerical columns really helps in this case, to provide additional information. I'll be sure to share any ideas for criteria that I have in the future. 

Thank you for your time, 
ToxicFyre
